trigger-openqa_in_openqa

```diff
@@ -23,7 +19,33 @@ client_prefix=${client_prefix:-}
 full_run=${FULL:-}
 group_id="${group_id:-"openQA"}"
 
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Triggers tests on an openQA instance testing openQA itself.
```

Comment:
```suggestion
Schedules tests on an openQA instance testing openQA itself.
Using the openQA-in-openQA test distribution from:
https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-openQA
```

Reply:
The application is called "trigger" so I would prefer to reuse that wording